pkg/init/cmd/service/system_init.go

```diff
@@ -78,8 +82,14 @@ func systemInitCmd(ctx context.Context, args []string) {
 	// remove (unlikely) old containerd socket
 	_ = os.Remove(*sock)
 
+	// look for containerd options
+	ctrdArgs := string{}
+	if b, err := ioutil.ReadFile(containerdOptsFile); err != nil {
+		ctrdArgs = strings.Fields(string(b))
```

Comment:
in theory you might want to support quoting so `--arg 'test value'` is not split, it might be worth at least adding a note about this, not sure Go has a shell-like parser you can use?

Reply:
I like the idea, but until I see an actual use case, I don't have the time to hunt something down. I will add a note about it in the faq.

Reply:
Updated the faq